applications/openshift/authentication/oauth_or_oauthclient_inactivity_timeout/policy/stig/shared.yml

```diff
@@ -0,0 +1,29 @@
+checktext: |-
+  To check if the OAuth server timeout is configured, run the following command:
+  >oc get oauth cluster -ojsonpath='{.spec.tokenConfig.accessTokenInactivityTimeout}'
+  If no number is displayed, or the timeout value is greater then the allowed idle timeout duration, this is a finding.
+
+  Determine if the session token inactivity timeout is set on the oauthclients by running the following.
+  > oc get oauthclients -ojsonpath='{range .items[*]}{.metadata.name}{"\t"}{.accessTokenInactivityTimeoutSeconds}{"\n"}'
+  The output will list each oauth client name followed by a number.  The number represents the timeout in seconds. If no number is displayed, or the timeout value is greater then the allowed idle timeout duration, this is a finding.
+
+fixtext: |-
+  To set the OAuth server inactivity timeout, edit the OAuth server
+  object: <tt>oc edit oauth cluster</tt>
+  and set the <tt>.spec.tokenConfig.accessTokenInactivityTimeout</tt>
+  parameter to the desired value:
+
+  apiVersion: config.openshift.io/v1
+  kind: OAuth
+  metadata:
+  ...
+  spec:
+     tokenConfig:
+       accessTokenInactivityTimeout: 10m0s 
+
+  For each oauth client that does not have the idle timeout set, or the timeout is set to the wrong duration, run the following command to set the idle timeout value to 10 minutes.
```

Comment:
This looks like the only fix text in the draft. Are lines 11 - 24 from somewhere else?